tuist/tuist
# Update xcbeautify to 0.13.0

### Short description 📝

XCBeautify has a new 0.13.0 release that fixes a sporadic hang: https://github.com/tuist/xcbeautify/releases/tag/0.13.0

When I tried to run `./fourier update xcbeautify` I found that `build_fat_release_binary` required an `xcode_paths` parameter now. So I added that to the `update` `xcbeautify` and `swiftformat` commands.

### How to test the changes locally 🧐

`./fourier update xcbeautify` and `./fourier update swiftformat` should work now


### Checklist ✅
